src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -153,19 +146,19 @@ const styles = StyleSheet.create({
 });
 
 type VehicleIconProps = {
-  vehicles: Coordinates;
+  coordinates: Coordinates;
   setShouldTrack: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
-const LiveVehicle = ({vehicles, setShouldTrack}: VehicleIconProps) => {
+const LiveVehicle = ({coordinates, setShouldTrack}: VehicleIconProps) => {
   const shapeSource = useRef<MapboxGL.ShapeSource>(null);
```

Comment:
Not changed in this commit, but what is the purpose of this `shapeSource` ref? 🤔

Reply:
That does look unnecessary, yes 🤔 Is it ok to remove @ingvisim?